apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,44 @@
 import { Component, OnInit } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CardType } from '../../model/card.model';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  templateUrl: 'city-card.component.html',
   standalone: true,
-  imports: [],
+  styles: [
+    `
+      ::ng-deep .bg-light-green {
```

Comment:
Try to avoid `::ng-deep` if not necessary, as in this challenge. With your implementation the class `bg-light-green` is declared globally, so if you use this class somewhere else in your application this styles will be assigned.